docs/concepts/scaling.md

```diff
@@ -0,0 +1,56 @@
+# Component Scaling
+
+TriggerMesh Components that are able to spin up replicas without leading to missing or duplicated events will scale uner heavy load.
+
+In contrast those components whose external system imposes a model where multiple instances of a client are not allowed or require distributed coordination won't scale..
+
+As a general rule scaling per component type defaults to:
+
+Type | Scalable | Exceptions
+------------ | ------------- | -------------
+Source | No | SlackSource, TwilioSource, WebhookSource and ZendeskSource
+Target | Yes |
+Transformation | Yes |
+Routing | Yes |
+
+!!! tip "Tip"
+    When a component that does not support scaling reachs its maximum capacity it is usually a good practice to partition the external service/data and configure multiple components.
+
+## Scaling behavior
+
+Scaling components are configured with the following parameters:
+
+- Scaler metrics: requests per second (RPS).
+- Minimum scale: 0, allows scale to zero.
+- Maximum scale: 30 instances.
+- Scale down delay: 5m, an scaled up instance will wait for 5m before being considered for retiring.
+- Requests per second: 300.
+- Time window: 1m, the window to aggregate metrics and take actions.
+
+As an example a load test for a single TriggerMesh component looks like this:
+
+!!! tip "Scale to 0"
+    Components that are able to scale are also able to scale to zero, saving resources when there is no load. A scaled to zero component can still receive data, that will be buffered by an activator agent while the backend is created (which usually takes ~ 1 second).
+
+    A component that has received no data for the last minute is a candidate for downscaling.
+
+!!! example "Example"
+    ![](../assets/images/scaling/scaling.png)
+
+    - Ramp up: the load for the first half of the test, the autoscaler creates replicas of the component based on demand.
+    - Stabilizing requests: there are some adjustments but the number of replicas is kept around 16 in the example.
+    - Ramp down: replicas start to be removed as long as they are not needed.
+    - Zero load: replicas are kept for 5 minutes receiving no requests, then they are removed.
+
+## Configuration
+
+When installing TriggerMesh scaling parameters can be customized to configure:
```

Comment:
how do i configure these parameters? honest question from me lol. but also one that I forsee a user asking when reading this

Reply:
[Here is the answer](https://knative.dev/docs/serving/autoscaling/).

Scaling parameters can be notchy, providing information about it is out of of scope for this doc (hence the PR name _user facing_). We are defaulting to the configuration that worked the better at our load tests, we cannot encourage users to use any other configuration unless they know well the internals of the autoscalers and how it fits their expected load.

Reply:
Agreed that this section seems like it could use a bit more info about how to go about this? Otherwise the section is a teaser about this possibility but doesn't give any direction as to how to tweak scaling settings. 

wdyt pablo?

Reply:
Did not expect these comments about the configuration section, maybe it would be better being removed.

At this post I don't want to push people to modify the recommended settings but understand how it works. That last section was added to inform that if there are scenarios with specific scaling needs, they can be tweaked, but parameters are up to Knative and would need load tests to make sure they fit.

Do you think it would be better without the `Configuration` section?